imap_processing/codice/packet_definitions/P_COD_HI_INST_COUNTS_AGGREGATED.xml

```diff
@@ -43,7 +43,7 @@
 				<xtce:UnitSet />
 				<xtce:BinaryDataEncoding bitOrder="mostSignificantBitFirst">
 					<xtce:SizeInBits>
-						<xtce:FixedValue>4325376</xtce:FixedValue>
+						<xtce:FixedValue>512</xtce:FixedValue>
```

Comment:
I changed a bunch of these with new hard-coded values to get it working with the simulated data that I have, but I intend to update these to use variable-length data in a future PR.